src/Files.App/UserControls/Pane/ShelfPane.xaml.cs

```diff
@@ -1,15 +1,102 @@
 // Copyright (c) Files Community
 // Licensed under the MIT License.
 
+using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
+using System.Runtime.InteropServices.ComTypes;
+using System.Windows.Input;
+using Vanara.PInvoke;
```

Comment:
Is `Vanara` supposed to be referenced?

Reply:
Yes, it's for the `ListView_DragItemsStarting` method (Just like `FileList_DragItemsStarting` references it too)

Reply:
Ideally we'd like to avoid adding references to Vanara but I think we can address this when we address the reference in `FileList_DragItemsStarting`.